regolith/helpers/u_institutionshelper.py

```diff
@@ -35,15 +35,15 @@ def subparser(subpi):
     subpi.add_argument("--dept_id",
                        help="e.g.: physics, apam.")
     subpi.add_argument("--dept_name",
-                       help="Department canonical name. e.g.: Department of Physics. "
+                       help="Department canonical name. e.g. Department of Physics. "
```

Comment:
actually the correct way to do these is:
```
"...canonical name, e.g., Department of Physics."
```

Reply:
Ok, sorry! I changed it!!